### PR TITLE
chore: separate out and disable keychain integration tests

### PR DESCRIPTION
**Pull request by mvayngrib**
## Summary

integration tests will be moved to the assets monorepo. For now, disabling here and removing relevant devDeps to decrease the load of open sourcing all dependencies in this repo

## Test Plan

- [ ] tests pass (wait for checks)
- [x] apply diff, run `yarn && cd features/keychain && yarn todo:reenable:test:integration` - tests pass
```diff
diff --git a/features/keychain/package.json b/features/keychain/package.json
index b0c1cae..d59faa5 100644
--- a/features/keychain/package.json
+++ b/features/keychain/package.json
@@ -45,6 +45,12 @@
     "p-defer": "^4.0.0"
   },
   "devDependencies": {
+    "@exodus/assets": "latest",
+    "@exodus/cardano-lib": "^2.0.3",
+    "@exodus/ethereum-lib": "^3.3.6",
+    "@exodus/solana-lib": "^1.3.11",
+    "@exodus/solana-meta": "^1.0.2",
+    "@exodus/storage-memory": "^2.1.1",
     "bip39": "2.6.0",
     "eslint": "^8.44.0",
     "events": "^3.3.0",

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
   "packageManager": "yarn@3.2.2",
   "dependencies": {
     "@exodus/migrate": "^1.5.3",
+    "delay": "^5.0.0",
```
Comment (mvayngrib) [inline package.json:93]: tmp, will update atoms in a separate PR https://github.com/ExodusMovement/exodus-hydra/pull/5307

File: package.json
Comment (diegomura) [inline package.json:1]: Sorry for the dumb question but how were these generated? Don't see changes in deps on package.json files

Comment (mvayngrib) [reply package.json:1]: that's my bad, accidentally committed them back 🤦

**Review by diegomura — APPROVED** [review]
utACK

File: package.json
Comment (mvayngrib) [reply package.json:1]: were u talking about keychain/package.json or the root?

File: features/keychain/module/__tests__/key-identifier.test.js [hunk 2]
```diff
@@ -9,61 +7,28 @@ describe('KeyIdentifier', () => {
...
       {},
       // Missing parameters
-      {
-        derivationAlgorithm: 'BIP32',
-        asset: assets.ethereum,
-      },
-      {
-        derivationPath: "m/44'/60'/0'/0/0",
-        asset: assets.ethereum,
-      },
```
Comment (mvayngrib) [inline features/keychain/module/__tests__/key-identifier.test.js:19]: moved to key-identifier.integration-test

File: package.json (hunk 1 above)
Comment (mvayngrib) [reply package.json:93]: updated in #16

**Review by feri42 — APPROVED** [review]
utACK

**Pull request merged**